hack/kustomize/test.sh

```diff
@@ -0,0 +1,84 @@
+#! /bin/bash
+
+# This script automates the functional testing of the collector
+
+DEFAULT_VERSION="1.2.7-beta1"
```

Comment:
Can we make these as arguments to this script (It already accepts cluster and token, and will also need these).

Reply:
Makes sense, we will add them